src/FsAutoComplete/FsAutoComplete.Lsp.fs

```diff
@@ -20,6 +20,11 @@ open FsToolkit.ErrorHandling
 
 module FcsRange = FSharp.Compiler.Range
 
+module AsyncResult =
```

Comment:
we have a pretty consistent pattern for returning non-Res CoreResponse values, so I've extracted that here so I can use `asyncResult`.